tests/run_pretrained_models.py

```diff
@@ -206,22 +215,21 @@ def run_test(self, name, backend="caffe2", onnx_file=None, opset=None, extra_ops
         input_names = list(self.input_names.keys())
         outputs = self.output_names
         if self.model_type in ["checkpoint"]:
-            graph_def, input_names, outputs = loader.from_checkpoint(model_path, input_names, outputs)
+            graph_def, input_names, outputs = tf_loader.from_checkpoint(model_path, input_names, outputs)
         elif self.model_type in ["saved_model"]:
-            graph_def, input_names, outputs = loader.from_saved_model(model_path, input_names, outputs)
+            graph_def, input_names, outputs = tf_loader.from_saved_model(model_path, input_names, outputs)
         else:
-            graph_def, input_names, outputs = loader.from_graphdef(model_path, input_names, outputs)
+            graph_def, input_names, outputs = tf_loader.from_graphdef(model_path, input_names, outputs)
 
-        # remove unused input names
-        input_names = list(set(input_names).intersection(self.input_names.keys()))
-        graph_def = tf2onnx.tfonnx.tf_optimize(input_names, self.output_names, graph_def, fold_const)
         if utils.is_debug_mode():
             utils.save_protobuf(os.path.join(TEMP_DIR, name + "_after_tf_optimize.pb"), graph_def)
```

Comment:
looks like the  tf_optimize() function is removed above, but the file name is still _after_tf_optimize.pb. The tf_optimize() was removed accidentally?

Reply:
tf_optimize is a few lines above inside the tf_session()